### ISSUE TITLE
Small issue when using a custom skin

### ISSUE THREAD

**Issue by eri-trabiccolo** [presscustomizr/customizr#540]
Notice:  Undefined index: custom-skin-xxxxx.css in xxx/wp-content/themes/customizr-pro/inc/class-fire-utils.phpon line 173
(with debug enabled, clearly)

This is due to the following to the following two lines:
https://github.com/presscustomizr/customizr/blob/v3.4.18/inc/class-fire-resources.php#L513
https://github.com/presscustomizr/customizr/blob/v3.4.18/inc/parts/class-content-slider.php#L1205

We didn't get the first earlier 'cause dropcap is disabled by default.
The second one clearly exposes the issue as it's called by default.

You can see why this happens here
-> https://github.com/presscustomizr/customizr/blob/v3.4.18/inc/class-fire-utils.php#L173

basically that code misses the check on the existence of the active skin key.
